code/modules/species/outsider/zombie.dm

```diff
@@ -417,7 +428,7 @@ GLOBAL_LIST_INIT(zombie_species, list(\
 		for(var/decl/hierarchy/skill/S in GLOB.skills) //Only want trained CQC and athletics
```

Comment:
```suggestion
		for(var/decl/hierarchy/skill/S in GLOB.skills) //Only want trained CQC and athletics
```

Reply:
Small note about this, you may want to update the comment here since you've upped their skills to experienced

Reply:
good shout